_pages/how-we-work/tools/google-calendar.md

```diff
@@ -14,38 +14,24 @@ TTS uses **shared calendars** for events, interviews, and "out of office" time.
+- [TTS Guilds and Working Groups](https://www.google.com/calendar/embed?src=gsa.gov_o1aqcv28k1f0nmca5bkch8los4%40group.calendar.google.com): Meetings and other events pertaining to working groups and guilds.
 
 **18F also uses these shared calendars:**
 
-* [18F Events](https://calendar.google.com/calendar/embed?src=gsa.gov_3rapmucstu32ma18da84el20ug%40group.calendar.google.com): Team-wide events, start dates, standups, and so on.
-* 18F Interviews: New hire interviews.
-* [18F Out of office](https://calendar.google.com/calendar/embed?src=gsa.gov_bth7useo0eeiicjgos2di6ph8k%40group.calendar.google.com): In addition to <em>officially</em> requesting time off in <a href="/gsa-internal-tools/#hr-links">HR Links</a>, please also add an event on the Google calendar "18F Out of Office" and invite yourself, so we all know when you’re gone.
+- [18F Events](https://calendar.google.com/calendar/embed?src=gsa.gov_3rapmucstu32ma18da84el20ug%40group.calendar.google.com): Team-wide events, start dates, standups, and so on.
+- 18F Interviews: New hire interviews.
+- [18F Out of office](https://calendar.google.com/calendar/embed?src=gsa.gov_bth7useo0eeiicjgos2di6ph8k%40group.calendar.google.com): In addition to <em>officially</em> requesting time off in <a href="/gsa-internal-tools/#hr-links">HR Links</a>, please also add an event on the Google calendar "18F Out of Office" and invite yourself, so we all know when you’re gone.
 
 You can add the above calendars to your Google calendar by clicking on the link to launch the HTML view and then clicking on `+Google Calendar` on the bottom right of the screen.
 
 ## Tips
 
-- To add a video call to your calendar invite: create an event, click **Edit event**, and under **Video call** click **Add video call**. Google gives the call a default name, but you can click **Change name** to customize it. Then click **Save**. ([Demonstration.]({{site.baseurl}}/images/google-calendar/2.gif))
+- [Add video calls to your calendar invites.](https://support.google.com/a/answer/4362302)
+- [Specify your working hours.](https://support.google.com/calendar/answer/7638168)
+- You can [view availability](https://support.google.com/calendar/answer/6294878) of other people at GSA for scheduling meetings.
 - You can't share calendars with anyone outside GSA. You can share individual invites, but not whole calendars.
-
-### Scheduling meetings with other people
-
-Here are a few ways you can use Google Calendar to find times to meet with other people:
-
-- To view someone else's calendar, type their name in the lefthand column under "other calendars."
-- Create an event, add everyone who needs to attend as a guest, and click "Suggested Times" or "Find a Time."
-- Set up blocks of time that people can reserve. Just select "Appointment slots" after you choose a window of time (this is especially useful for researchers).
-
-### Specify working hours
```

Comment:
This seems like a tip worth keeping, maybe?

Reply:
Yep! I changed it to a link above.